### PR TITLE
Host bio

### PR DESCRIPTION
**Pull request by HassenH1**

### CODE REVIEW

File: app/src/components/HostFormDetail/HostFormGender.tsx [hunk 1]
```diff
@@ -0,0 +1,288 @@
+import React, { useState } from 'react'
```
Comment (tylerthome) [inline app/src/components/HostFormDetail/HostFormGender.tsx:1]: @HassenH1 for the JSX elements using identical style attributes please define these variables of type `CSSProperties` 

For example, one repeated rule set apparent in this snippet could be factored out into the following reusable object:
```
const fieldContainerStyles: CSSProperties = {
   display: 'flex',
   flexDirection: 'row',
   alignItems: 'center',
   fontSize: '18px',
}
```

and used in the JSX as `<div style={fieldContainerStyles}></div>` where applicable

Comment (HassenH1) [reply app/src/components/HostFormDetail/HostFormGender.tsx]: Just added this

**Comment by HassenH1** [comment]
Actually, Dont merge this
I forgot to add something

**Comment by HassenH1** [comment]
Ready to Go

**Pull request merged**